authenticator/src/main/kotlin/com/bitwarden/authenticator/data/platform/datasource/disk/SettingsDiskSourceImpl.kt

```diff
@@ -1,29 +1,29 @@
 package com.bitwarden.authenticator.data.platform.datasource.disk
 
 import android.content.SharedPreferences
-import com.bitwarden.authenticator.data.platform.datasource.disk.BaseDiskSource.Companion.BASE_KEY
 import com.bitwarden.authenticator.ui.platform.feature.settings.appearance.model.AppLanguage
 import com.bitwarden.authenticator.ui.platform.feature.settings.appearance.model.AppTheme
 import com.bitwarden.authenticator.ui.platform.feature.settings.data.model.DefaultSaveOption
 import com.bitwarden.core.data.repository.util.bufferedMutableSharedFlow
+import com.bitwarden.data.datasource.disk.BaseDiskSource
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.onSubscription
 
-private const val APP_THEME_KEY = "$BASE_KEY:theme"
-private const val APP_LANGUAGE_KEY = "$BASE_KEY:appLocale"
-private const val DEFAULT_SAVE_OPTION_KEY = "$BASE_KEY:defaultSaveOption"
-private const val SYSTEM_BIOMETRIC_INTEGRITY_SOURCE_KEY = "$BASE_KEY:biometricIntegritySource"
-private const val ACCOUNT_BIOMETRIC_INTEGRITY_VALID_KEY = "$BASE_KEY:accountBiometricIntegrityValid"
-private const val ALERT_THRESHOLD_SECONDS_KEY = "$BASE_KEY:alertThresholdSeconds"
-private const val FIRST_LAUNCH_KEY = "$BASE_KEY:hasSeenWelcomeTutorial"
-private const val CRASH_LOGGING_ENABLED_KEY = "$BASE_KEY:crashLoggingEnabled"
+private const val APP_THEME_KEY = "theme"
+private const val APP_LANGUAGE_KEY = "appLocale"
+private const val DEFAULT_SAVE_OPTION_KEY = "defaultSaveOption"
+private const val SYSTEM_BIOMETRIC_INTEGRITY_SOURCE_KEY = "biometricIntegritySource"
+private const val ACCOUNT_BIOMETRIC_INTEGRITY_VALID_KEY = "accountBiometricIntegrityValid"
+private const val ALERT_THRESHOLD_SECONDS_KEY = "alertThresholdSeconds"
+private const val FIRST_LAUNCH_KEY = "hasSeenWelcomeTutorial"
+private const val CRASH_LOGGING_ENABLED_KEY = "crashLoggingEnabled"
```

Comment:
`BASE_KEY` was inlined in `:authenticator` but applied to keys within `BaseDiskSource` in the `:app` module. Prefixes were removed from the `:authenticator` so they are not applied twice when invoking `BaseDiskSource`.